### ISSUE TITLE
User management 

### ISSUE THREAD

**Issue by numahell** [GeotrekCE/Geotrek-admin#3709]
About user management documentation, there is a part in english in installation section, and another part in french in user manual.

- https://geotrek.readthedocs.io/en/latest/install/configuration.html#users-management
- https://geotrek.readthedocs.io/en/latest/usage/configuration-ttw.html#gestion-des-utilisateurs

I think it would be better to merge all user management stuff in user manual, and use translation. To avoid broken link, I suggest to keep title in install section with a link to user manual.

**Comment by babastienne** [comment]
I agree with you. I'm not sure tho about keeping the title in the install section. Removing it will create a broken link indeed but i'm not sure it is an important breaking change, not important in my opinion
